staticcheck/lint.go

```diff
@@ -3926,20 +3923,28 @@ func checkJSONTag(pass *analysis.Pass, field *ast.Field, tag string) {
 					}
 				}
 			}
-		case "inline":
-			ci++
+			options[s] = options[s] + 1
```

Comment:
In the old code we increment `cs` at the top, so that we increment it even if we hit the `continue`. We should maintain that property,

Reply:
Done.